main.go

```diff
@@ -20,11 +30,98 @@ type Stream struct {
 func main() {
 	apiAddr := os.Getenv("OPENSHIFT_API_URL")
 	apiToken := os.Getenv("OPENSHIFT_TOKEN")
+	syslogServer := os.Getenv("SYSLOG_SERVER")
+	syslogProto := strings.ToLower(os.Getenv("SYSLOG_PROTO"))
+	syslogTag := strings.ToUpper(os.Getenv("SYSLOG_TAG"))
+	ignoreSSL := strings.ToUpper(os.Getenv("IGNORE_SSL"))
+	debugFlag := strings.ToUpper(os.Getenv("DEBUG"))
+
+	// enable signal trapping
+	go func() {
+		c := make(chan os.Signal, 1)
+		signal.Notify(c,
+			syscall.SIGINT,  // Ctrl+C
+			syscall.SIGTERM, // Termination Request
+			syscall.SIGSEGV, // FullDerp
+			syscall.SIGABRT, // Abnormal termination
+			syscall.SIGILL,  // illegal instruction
+			syscall.SIGFPE)  // floating point
+		sig := <-c
+		log.Fatalf("Signal (%v) Detected, Shutting Down", sig)
+	}()
+
+	// check and make sure we have the minimum config information before continuing
```

Comment:
typo

Reply:
I am not seeing the typo (probably why it is there ... can you explain what I am missing... )